webdriver/webapp_server.go

```diff
@@ -132,17 +132,17 @@ func NewDevAppServer() (s DevAppServerInstance, err error) {
 	i := &devAppServerInstance{
 		startupTimeout: 15 * time.Second,
 
-		host:      "localhost",
-		port:      8080,
-		adminPort: 8000,
-		apiPort:   9999,
+		host:    "localhost",
+		port:    pickUnusedPort(),
+		apiPort: pickUnusedPort(),
 	}
 
 	i.cmd = exec.Command(
 		"dev_appserver.py",
 		fmt.Sprintf("--port=%d", i.port),
-		fmt.Sprintf("--admin_port=%d", i.adminPort),
 		fmt.Sprintf("--api_port=%d", i.apiPort),
+		// Let dev_appserver find a free port itself.
```

Comment:
Was this change to reduce the number of things we have to specify in go (using `pickUnusedPort()`) or is there a technical reason that we _need_ to let dev_appserver pick the port? If it's the latter, that reason should probably be added to this comment. Otherwise, ignore me.

Reply:
It was mainly to reduce the likelihood of race conditions, as dev_appserver will bind to the free port given by the kernel directly. I'll make this clearer in the comment.